### ISSUE TITLE
Anti-DREBRIN antibody positivity

### ISSUE THREAD

**Issue by MaudDLVT** [obophenotype/human-phenotype-ontology#10505]
**Preferred term label:**
Anti-DREBRIN antibody positivity

**Synonyms**

**Definition (free text, please give PubMed ID)**
The presence of autoantibodies in the serum that react against dendritic spine scaffolding protein Drebrin
PMID: 32196746

**Parent term (use hpo.jax.org/app)**
HP:0030057

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:624166

**Your nano-attribution (ORCID)**
0000-0002-3387-1836

**Comment by pnrobinson** [comment]
@MaudDLVT 
Adding new term: 
- ID: HP:6001059
- label: Anti-DREBRIN antibody positivity
- parents: [Autoimmune antibody positivity (HP:0030057)](https://hpo.jax.org/app/browse/term/HP:0030057)
- pmid: PMID:32196746
- def: The presence of autoantibodies in the serum that react against dendritic spine scaffolding protein Drebrin.
- comment: 
- orcid: orcid:0000-0002-3387-1836
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10505
- synonyms:
